maintain/scale-up.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Adding storage to existing PX Cluster Nodes"
 keywords: scale-up
 sidebar: home_sidebar
-redirect_from: "/scale-up.html"
+redirect_from:
+  - /scale-up.html
+  - /convert-x86-to-storage-nodes.html
```

Comment:
What was this page? Does it belong here?

Reply:
It was actually this file https://github.com/portworx/px-docs/blob/dcf11db845f120eb11f2df503e08f7dc09c03730/run-with-docker.md

So can we move this to the https://docs.portworx.com/scheduler/docker/install-standalone.html page?